.github/workflows/Tests.yaml

```diff
@@ -25,32 +30,43 @@ jobs:
       - name: Set up Python ${{ matrix.python }}
         uses: actions/setup-python@v4
         with:
-          python-version: "3.11"
-          cache: "pipenv"
+          python-version: ${{ matrix.python }}
+          architecture: x64
+          cache: 'pip'
 
-      - name: Setup dependencies.
+      - name: Install dependencies (and project)
         working-directory: backend
         run: |
-          pip install --upgrade pipenv
-          pipenv sync
+          pip install -U pip
+          pip install -e .[test,scripts]
 
-      - name: Setup database.
-        working-directory: backend
-        run: |
-          pipenv run init
+      - name: Initialize the database
+        working-directory: backend/src
         env:
           POSTGRES_URI: "postgresql+psycopg://nautilus:nautilus@localhost:5432/nautilus"
+        run: alembic upgrade head
 
-      - name: Run tests
+      - name: Run the tests
         working-directory: backend
-        run: |
-          pipenv run invoke test --args "-v"
-          pipenv run invoke generate-xml-report
         env:
           POSTGRES_URI: "postgresql+psycopg://nautilus:nautilus@localhost:5432/nautilus"
+        run: inv coverage --args "-vvv"
+
+      - name: Generate the coverage report
```

Comment:
I just updated it😂.
There is a subtle difference between codedev's `root_dir` and `working-directory`. The right way is setting the `root_dir` instead of the `working-directory`.